hiss/static/main.js

```diff
@@ -23,4 +23,17 @@ $(function() {
              $('#id_gender_other').parent().hide();
          }
     });
+
+    if ($('#id_school option:selected').text() !== "Other"){
+        $('#id_school_other').parent().hide();
+    }
+    $('#id_school').on('change', function(){
+         let selection = $('#id_school option:selected').text();
+         if (selection === "Other"){
+            $('#id_school_other').parent().show();
+         }
+         else{
+             $('#id_school_other').parent().hide();
+         }
+    });
```

Comment:
This is a hack, I don't want to bind the "Other" option to a specific value in the fixtures because that's not really generalizable. But I'm not sure how to make this more general. Thoughts?

Reply:
I think this is the best way of doing this for now.